schunk_lwa4d_moveit_config/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(lwa4d_moveit_config)
```

Comment:
Well I renamed the file and now I realise that it was not a good option. What do you recommend to do now?